tests/conftest.py

```diff
@@ -24,13 +24,16 @@ def file_path() -> str:
 
 @pytest.fixture
 def args() -> Namespace:
-    return Namespace(path=["tests/samples/password"], api_validation="true", json_filename=None)
+    dir_path = os.path.dirname(os.path.realpath(__file__))
+    file_name = os.path.join(dir_path, "samples", "password")
+    return Namespace(path=[file_name], api_validation="true", json_filename=None)
 
 
 @pytest.fixture
 def config() -> Config:
     dir_path = os.path.dirname(os.path.realpath(__file__))
```

Comment:
How about moving `dir_path` to the file level or a separate fixture? As it seems to be used in multiple places with now